doc/src/pair_aip_water_2dm.rst

```diff
@@ -0,0 +1,165 @@
+.. index:: pair_style aip/water/2dm
+.. index:: pair_style aip/water/2dm/opt
+
+pair_style aip/water/2dm command
+===================================
+
+Accelerator Variant: *aip/water/2dm/opt*
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style [hybrid/overlay ...] aip/water/2dm cutoff tap_flag
+
+* cutoff = global cutoff (distance units)
+* tap_flag = 0/1 to turn off/on the taper function
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style  hybrid/overlay aip/water/2dm 16.0 1
+   pair_coeff  * * aip/water/2dm  COH.aip.water.2dm C Ow Hw
+
+   pair_style  hybrid/overlay aip/water/2dm 16.0 lj/cut/tip4p/long 2 3 1 1 0.1546 10 8.5
+   pair_coeff  2 2   lj/cut/tip4p/long    8.0313e-3  3.1589  # O-O
+   pair_coeff  2 3   lj/cut/tip4p/long    0.0        0.0     # O-H
+   pair_coeff  3 3   lj/cut/tip4p/long    0.0        0.0     # H-H
+   pair_coeff  * *   aip/water/2dm        COH.aip.water.2dm    C Ow Hw
+
+Description
+"""""""""""
+
+.. versionadded:: xxxx2023
+
+The *aip/water/2dm* style computes the anisotropic interfacial
+potential (AIP) potential for interfaces of water with two-dimensinal (2D)
+materials as described in :ref:`(Feng) <Feng>`.
+
+.. math::
+
+   E  = & \frac{1}{2} \sum_i \sum_{j \neq i} V_{ij} \\
+   V_{ij}  = & {\rm Tap}(r_{ij})\left \{ e^{-\alpha (r_{ij}/\beta -1)}
+                \left [ \epsilon + f(\rho_{ij}) + f(\rho_{ji})\right ] -
+                 \frac{1}{1+e^{-d\left [ \left ( r_{ij}/\left (s_R \cdot r^{eff} \right ) \right )-1 \right ]}}
+                 \cdot \frac{C_6}{r^6_{ij}} \right \}\\
+   \rho_{ij}^2 = & r_{ij}^2 - ({\bf r}_{ij} \cdot {\bf n}_i)^2 \\
+   \rho_{ji}^2  = & r_{ij}^2 - ({\bf r}_{ij} \cdot {\bf n}_j)^2 \\
+   f(\rho)  = &  C e^{ -( \rho / \delta )^2 } \\
+   {\rm Tap}(r_{ij})  = & 20\left ( \frac{r_{ij}}{R_{cut}} \right )^7 -
+                           70\left ( \frac{r_{ij}}{R_{cut}} \right )^6 +
+                           84\left ( \frac{r_{ij}}{R_{cut}} \right )^5 -
+                           35\left ( \frac{r_{ij}}{R_{cut}} \right )^4 + 1
+
+Where :math:`\mathrm{Tap}(r_{ij})` is the taper function which provides
+a continuous cutoff (up to third derivative) for interatomic separations
+larger than :math:`r_c` :doc:`pair_style ilp_graphene_hbn <pair_ilp_graphene_hbn>`.
+
+It is important to include all the pairs to build the neighbor list for
```

Comment:
"include all the pairs" of what?
What neighbor list does this refer to?
Does this mean that a user may need to use a particular "special_bonds" setting (e.g. special_bonds lj/cut 1.0e-100 1.0e-100 1.0e-100) for pair styles that do not require Kspace?

Reply:
To calculate the atomic normal vectors, a full neighbor list is required. In the case of a hydrogen atom, its normal vector can be determined by identifying its neighboring oxygen atom and applying the following definition: the normal vector of a hydrogen atom is assumed to lie along the corresponding oxygen-hydrogen bond.

Since it belongs to the detail of the codes, this sentence can be deleted here.